movies.py

```diff
@@ -0,0 +1,82 @@
+"""Movie ticket process system and database"""
+
+MOVIE_PRICE = 5.0
+
+def enough_money(customer):
+    """Check if customer has adequate funds"""
+    balance = customer["cash"]
+    if balance < MOVIE_PRICE:
+        return False
+    return True
+
+def purchase_ticket(movie, customer):
+    """Performs the steps required in purchasing a ticket and returns altered customer and movie"""
+    assert enough_money(customer)
+    customer = charge(customer)
+    customer = dispense_ticket(movie, customer)
```

Comment:
This improves the readability by making it more concise.
i.e. - "dispense the ticket for the movie and charge the customer"